src/main/groovy/net/nemerosa/versioning/git/GitInfoService.groovy

```diff
@@ -19,7 +19,10 @@ class GitInfoService implements SCMInfoService {
     @Override
     SCMInfo getInfo(Project project, VersioningExtension extension) {
         // Is Git enabled?
-        boolean hasGit = project.rootProject.file('.git').exists()
+        boolean hasGit = project.rootProject.file('.git').exists() ||
+                         project.file('.git').exists() ||
+                         (extension.gitRepoRootDir != null &&
+                                 new File("${extension.gitRepoRootDir}${File.pathSeparator}.git").exists())
```

Comment:
Actually, this _has_ to be change. `File.pathSeparator` is not correct ; at best, it should be changed to `File.separator`.

FYI - I'm defined a test on the `domcc-ISSUE-37` branch.